p2panda-js/src/wasm.test.ts

```diff
@@ -99,6 +99,22 @@ describe('WebAssembly interface', () => {
         'Invalid integer value',
       );
 
+      expect(() => fields.add('contact', 'relation', [TEST_HASH])).toThrow(
+        'Invalid object',
```

Comment:
I think this error message could be improved

Reply:
https://github.com/p2panda/p2panda/issues/262

Reply:
Yeh, very true, you wanna do that as part of this PR or a separate one?

Reply:
No separately!

Reply:
I see in the issue I think you wanna do a wider revamp of the error messages :+1: